### PR TITLE
PIM-5241: Add css for weird multiselect display

### PR DESCRIPTION
**Pull request by ctreton**
| Q                 | A
| ----------------- | ---
| Specs             | y
| Behats            | n
| Blue CI           | y
| Changelog updated | y
| Review and 2 GTM  | y

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/css/mass-edit.less [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .edit-common-attributes {
   min-height: 400px;
 }
+.ui-multiselect-filter input {
+  box-sizing: content-box;
+}
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Resources/public/css/mass-edit.less]: missing new line

File: CHANGELOG-1.4.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 - PIM-5171: Apply ACLs on mass edit actions
 - PIM-5240: Fix Mongo normalization that creates a nullable family field
 - PIM-5159: Attribute values appearing/disappearing when you change attribute groups
+- PIM-5241: Fix search input field with strange display on Firefox / Mass Edit on Common Attributes
```
Comment (rybus) [inline CHANGELOG-1.4.md]: common attributes

File: src/Pim/Bundle/EnrichBundle/Resources/public/css/mass-edit.less [hunk 3]
```diff
@@ -1,3 +1,6 @@
 .edit-common-attributes {
   min-height: 400px;
 }
+.ui-multiselect-filter input {
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Resources/public/css/mass-edit.less]: maybe you can add that in pim.less ?

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/css/mass-edit.less]: yup should be on pim.less

**Comment by phaseinducer** [comment]
GTM

**Comment by pierallard** [comment]
GTM

**Comment by skeleton** [comment]
GTM

**Comment by ClementGautier** [comment]
GTM

**Comment by nidup** [comment]
thank you @ctreton !

**Pull request merged**